packages/vm/test/retesteth/transition-child.ts

```diff
@@ -125,7 +125,7 @@ async function runTransition(argsIn: any) {
   const logsBloom = builder.logsBloom()
   const logsHash = keccak256(logsBloom)
 
-  await vm.stateManager.cleanupTouchedAccounts()
+  await vm.evm.evmJournal.cleanupTouchedAccounts()
```

Comment:
Shouldn't this just be `evmJournal.cleanup()`?

Reply:
Yup, `cleanupTouchedAccounts` is not part of the interface anymore. Good catch.

Reply:
Thank you VS Code.